### ISSUE TITLE
Document using Docker container locally

### ISSUE THREAD

**Issue by hturner** [r-devel/r-dev-env#51]
One issue is that there are probably several variations of this: with/without VS Code, with/without Docker Desktop and potentially different instructions for different OS.

In addition, this is not a good route for people that are new to Docker as it requires a lot of set up in itself, e.g. requiring WSL on Windows.

So we should think carefully about what we want to "support" via documentation - something like a quick start guide for people that are already set up and familiar with Docker, pointing to external docs where possible.

**Comment by jamestripp** [comment]
Good point. It may be enough to point out that it can work but is not 'supported'. I would be happy to put together a paragraph or two linking to external resources.

**Comment by hturner** [comment]
Perhaps ask @eliocamp if he would like to test this out once you've added the documentation.

**Comment by eliocamp** [comment]
It works really well. Kind of like magic!
